src/app.js

```diff
@@ -4,38 +4,69 @@ const GitHubApi = require('github');
 const nacl = require('tweetnacl');
 nacl.util = require('tweetnacl-util');
 
-const username = 'yourusername';  // TODO: your GitHub username here
+require('dotenv').config();
+
+const username = 'tacolim';  // TODO: your GitHub username here
 const github = new GitHubApi({ debug: true });
 const server = express();
+const token = process.env.GITHUB_TOKEN;
+let clientID = '';
+console.log('token:', token);
+
+server.use(bodyParser.json());
 
 // Generate an access token: https://github.com/settings/tokens
 // Set it to be able to create gists
 github.authenticate({
   type: 'oauth',
-  token: process.env.GITHUB_TOKEN
+  token
 });
 
 // Set up the encryption - use process.env.SECRET_KEY if it exists
 // TODO either use or generate a new 32 byte key
+const key = process.env.SECRET_KEY ? nacl.util.decodeBase64(process.env.SECRET_KEY) : nacl.randomBytes(32);
 
 server.get('/', (req, res) => {
   // TODO Return a response that documents the other routes/operations available
+  github.users.getForUser({ username })
+    .then((response) => {
+      res.json(response.data);
+      clientID = response.data.id;
+    }).catch((err) => { throw new Error('Error', err.message) })
```

Comment:
Is there a better way to handle errors in this case?

Reply:
One approach to consider - logging. There are different logging levels too, we've not talked about it much but it can help organize these things.